requirements/requirements.txt

```
@@ -10,9 +10,9 @@ id-marl-eval @ git+https://github.com/instadeepai/marl-eval
 jax==0.4.30
 jaxlib==0.4.30
 jaxmarl @ git+https://github.com/RuanJohn/JaxMARL@unpin-jax  # This only unpins the version of Jax.
-jumanji @ git+https://github.com/sash-a/jumanji@old_jumanji  # Includes a few extra MARL envs
+jumanji @ git+https://github.com/instadeepai/jumanji
 lbforaging
-matrax @ git+https://github.com/instadeepai/matrax@4c5d8aa97214848ea659274f16c48918c13e845b
+matrax @ git+https://github.com/instadeepai/matrax
```

Similarly for matrax, although I think we still need to release that one
```suggestion
matrax
```

---

Yep with the newer version since now it has 0.0.4 which depends on old jumanji this will be an issue :/
Can we do that or Arnu should?